src/sas/sascalc/dataloader/data_info.py

```diff
@@ -92,14 +39,17 @@ class plottable_1D(object):
     dxl = None
     ## Slit smearing width
     dxw = None
+    ## SESANS specific params (wavelengths for spin echo length calculation)
+    lam = None
+    dlam = None

```

Comment:
If it is a placeholder for future development I would argue It should stay but should be well documented in comments why it is there so that if plans change somebody else down the road knows it can safely be deleted.